packages/graph/migrations/postgres/migration/1656417312397_initial.ts

```diff
@@ -0,0 +1,209 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { MigrationBuilder, ColumnDefinitions } from "node-pg-migrate";
+import { stripNewLines } from "../util";
+
+export const shorthands: ColumnDefinitions | undefined = undefined;
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.createTable(
+    "accounts",
+    {
+      account_id: {
+        type: "UUID",
+        primaryKey: true,
+        comment:
+          "Accounts are undecided and just here for satisfying the future schema",
+      },
+    },
+    {
+      ifNotExists: true,
+    },
+  );
+
+  pgm.createTable(
+    "ids",
+    {
+      version_id: {
+        type: "UUID",
+        primaryKey: true,
+      },
+      id: {
+        type: "UUID",
+        notNull: true,
+        unique: true,
```

Comment:
Should the `id` be unique? Shouldn't the `version_id` be unique?

Reply:
> Should the id be unique?
Good point, more than one `version_id` can have the same `id` - have addressed in https://github.com/hashintel/hash/pull/712/commits/692832839cea317a0ad4ecf08ecb6ed97343adef

Reply:
Gotcha, my lack of knowledge shining through :D